test/unit/specs/components/v-content-link.spec.js

```diff
@@ -0,0 +1,48 @@
+import VContentLink from '~/components/VContentLink/VContentLink.vue'
+import { render, screen } from '@testing-library/vue'
+import userEvent from '@testing-library/user-event'
+import VueI18n from 'vue-i18n'
+
+const enMessages = require('~/locales/en.json')
+const i18n = new VueI18n({
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages: { en: enMessages },
+})
+
+describe('VContentLink', () => {
+  let options = {}
+
+  beforeEach(() => {
+    options = {
+      props: { mediaType: 'image', resultsCount: 123 },
+      mocks: {
+        $nuxt: {
+          context: {
+            i18n,
+          },
+        },
+      },
+    }
+  })
+
+  it('is not selected by default', () => {
+    render(VContentLink, options)
+    const btn = screen.getByRole('radio')
+    expect(btn).not.toHaveAttribute('aria-checked')
+  })
+
+  it('is marked as selected when indicated with the isSelected prop', () => {
+    options.props.isSelected = true
+    render(VContentLink, options)
+    const btn = screen.getByRole('radio')
+    expect(btn).toHaveAttribute('aria-checked')
```

Comment:
For some extra assurance it might be good to also check the value of `aria-checked` is `"true"`. I suppose it could also be `"false"` (or any other arbitrary value) and this test would pass.

Reply:
I forgot it, thanks! Added the check in [`dbe772a`](https://github.com/WordPress/openverse-frontend/pull/560/commits/dbe772ae272177968abcce54e1d5138d7383eb1a).